src/test/cdk/explorer/constructNode.test.ts

```diff
@@ -94,18 +98,53 @@ describe('ConstructNode', () => {
             children: childWithAttributes
         }
 
-        const testNode = new ConstructNode(label, vscode.TreeItemCollapsibleState.Collapsed, treeEntity)
+        const testNode = new ConstructNode(label, vscode.TreeItemCollapsibleState.Collapsed, treeEntity, treeFsPath)
 
         const childNodes = await testNode.getChildren()
         assert.strictEqual(childNodes.length, 1)
         assert.strictEqual(childNodes[0].collapsibleState, vscode.TreeItemCollapsibleState.Collapsed)
     })
 
+    it('returns child node of PropertyNode when construct has props', async () => {
+        const treeEntity: ConstructTreeEntity = {
+            id: label,
+            path: path,
+            attributes: treeUtils.generateAttributes()
+        }
+
+        const testNode = new ConstructNode(label, vscode.TreeItemCollapsibleState.Collapsed, treeEntity, treeFsPath)
+
+        const childNodes = await testNode.getChildren()
+        assert.strictEqual(childNodes.length, 1)
+        assert.strictEqual(childNodes[0] instanceof PropertyNode, true)
```

Comment:
These kind of tests are difficult to understand a failure at a glance. It would report "expected true, got false".
We're trying to do something like this now:
`assert.ok(childNodes[0] instanceof PropertyNode, 'Expected child node to be a PropertyNode')`
Which you can then immediately tell why the test failed.

(This applies elsewhere in the PR too)

Reply:
fair point. I do have a hard time finding the balance between "it's covered in the test name", and "this assertion is too msytical"

tried to go in and add more descriptive messages where I felt it was needed